csharp/Ice/latency/server/Server.cs

```diff
@@ -8,7 +8,7 @@
 using var communicator = new Communicator(ref args, ConfigurationManager.AppSettings);
 
 // Destroy the communicator on Ctrl+C or Ctrl+Break
```

Comment:
The comment needs to be fixed, e.g.:
```
// Initiates communicator shutdown on Ctrl+C or Ctrl+Break, but does not wait for shutdown to complete.
```

Except for dispose vs shutdown, it's just like in the client, with the same "start but don't wait for completion" behavior. We use async/await in the client to avoid a warning because DisposeAsync returns a value task. ShutdownAsync returns a Task so no warning and no need for async/await.

Reply:
Then further down, it should be:
```
await adapter.ActivateAsync();
await communicator.WaitForShudownAync();
```